<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-QFKR4O5H" ignored="0" name="Identify Courses in Firefox" tests="20" failures="7" timestamp="05 Oct 2020 08:04:04 GMT" time="550.167" errors="0">
  <testcase name="searchTest" time="109.004" classname="TestCases.IdentifyCoursesTest"/>
  <testcase name="formTest" time="21.055" classname="TestCases.IdentifyCoursesTest"/>
  <testcase name="formTest" time="24.828" classname="TestCases.IdentifyCoursesTest"/>
  <testcase name="formTest" time="25.617" classname="TestCases.IdentifyCoursesTest"/>
  <testcase name="formTest" time="21.905" classname="TestCases.IdentifyCoursesTest"/>
  <testcase name="formTest" time="19.797" classname="TestCases.IdentifyCoursesTest"/>
  <testcase name="formTest" time="19.489" classname="TestCases.IdentifyCoursesTest"/>
  <testcase name="formTest" time="74.717" classname="TestCases.IdentifyCoursesTest">
    <failure type="java.lang.AssertionError" message="Error message NOT displayed, which is a failure!">
      <![CDATA[java.lang.AssertionError: Error message NOT displayed, which is a failure!
at BaseClasses.BaseTestClass.reportFail(BaseTestClass.java:174)
at PageClasses.CourseraForCampusPage.fillAndSubmitForm(CourseraForCampusPage.java:117)
at TestCases.IdentifyCoursesTest.formTest(IdentifyCoursesTest.java:66)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- formTest -->
  <testcase name="formTest" time="17.651" classname="TestCases.IdentifyCoursesTest">
    <failure type="java.lang.AssertionError" message="Error message NOT displayed, which is a failure!">
      <![CDATA[java.lang.AssertionError: Error message NOT displayed, which is a failure!
at BaseClasses.BaseTestClass.reportFail(BaseTestClass.java:174)
at PageClasses.CourseraForCampusPage.fillAndSubmitForm(CourseraForCampusPage.java:117)
at TestCases.IdentifyCoursesTest.formTest(IdentifyCoursesTest.java:66)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- formTest -->
  <testcase name="formTest" time="17.453" classname="TestCases.IdentifyCoursesTest">
    <failure type="java.lang.AssertionError" message="Error message NOT displayed, which is a failure!">
      <![CDATA[java.lang.AssertionError: Error message NOT displayed, which is a failure!
at BaseClasses.BaseTestClass.reportFail(BaseTestClass.java:174)
at PageClasses.CourseraForCampusPage.fillAndSubmitForm(CourseraForCampusPage.java:117)
at TestCases.IdentifyCoursesTest.formTest(IdentifyCoursesTest.java:66)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- formTest -->
  <testcase name="formTest" time="18.605" classname="TestCases.IdentifyCoursesTest">
    <failure type="java.lang.AssertionError" message="Error message NOT displayed, which is a failure!">
      <![CDATA[java.lang.AssertionError: Error message NOT displayed, which is a failure!
at BaseClasses.BaseTestClass.reportFail(BaseTestClass.java:174)
at PageClasses.CourseraForCampusPage.fillAndSubmitForm(CourseraForCampusPage.java:117)
at TestCases.IdentifyCoursesTest.formTest(IdentifyCoursesTest.java:66)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- formTest -->
  <testcase name="formTest" time="19.194" classname="TestCases.IdentifyCoursesTest">
    <failure type="java.lang.AssertionError" message="Error message NOT displayed, which is a failure!">
      <![CDATA[java.lang.AssertionError: Error message NOT displayed, which is a failure!
at BaseClasses.BaseTestClass.reportFail(BaseTestClass.java:174)
at PageClasses.CourseraForCampusPage.fillAndSubmitForm(CourseraForCampusPage.java:117)
at TestCases.IdentifyCoursesTest.formTest(IdentifyCoursesTest.java:66)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- formTest -->
  <testcase name="formTest" time="19.815" classname="TestCases.IdentifyCoursesTest">
    <failure type="java.lang.AssertionError" message="Error message NOT displayed, which is a failure!">
      <![CDATA[java.lang.AssertionError: Error message NOT displayed, which is a failure!
at BaseClasses.BaseTestClass.reportFail(BaseTestClass.java:174)
at PageClasses.CourseraForCampusPage.fillAndSubmitForm(CourseraForCampusPage.java:117)
at TestCases.IdentifyCoursesTest.formTest(IdentifyCoursesTest.java:66)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- formTest -->
  <testcase name="formTest" time="17.958" classname="TestCases.IdentifyCoursesTest">
    <failure type="java.lang.AssertionError" message="Error message NOT displayed, which is a failure!">
      <![CDATA[java.lang.AssertionError: Error message NOT displayed, which is a failure!
at BaseClasses.BaseTestClass.reportFail(BaseTestClass.java:174)
at PageClasses.CourseraForCampusPage.fillAndSubmitForm(CourseraForCampusPage.java:117)
at TestCases.IdentifyCoursesTest.formTest(IdentifyCoursesTest.java:66)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- formTest -->
  <testcase name="formTest" time="20.639" classname="TestCases.IdentifyCoursesTest"/>
  <testcase name="formTest" time="19.202" classname="TestCases.IdentifyCoursesTest"/>
  <testcase name="formTest" time="16.655" classname="TestCases.IdentifyCoursesTest"/>
  <testcase name="formTest" time="16.798" classname="TestCases.IdentifyCoursesTest"/>
  <testcase name="formTest" time="19.072" classname="TestCases.IdentifyCoursesTest"/>
  <testcase name="formTest" time="25.587" classname="TestCases.IdentifyCoursesTest"/>
</testsuite> <!-- Identify Courses in Firefox -->
